sdcm/nemesis.py

```diff
@@ -1560,17 +1559,19 @@ def disrupt_nodetool_refresh(self, big_sstable: bool = False):
             result = self.target_node.run_cqlsh(query_verify)
             assert '(1 rows)' in result.stdout, f'The key {key} is not loaded by `nodetool refresh`'
 
-    def disrupt_nodetool_enospc(self, sleep_time=30, all_nodes=False):
-        if isinstance(self.cluster, LocalKindCluster):
-            # Kind cluster has shared file system, it is shared not only among cluster nodes, but
-            # also among kubernetes services which make kubernetes inoperational once enospc is reached.
-            # Moreover, it will cause host system's /var hosting disk go out of free space too
-            # which may cause unpredictable failures.
-            raise UnsupportedNemesis('disrupt_nodetool_enospc is not supported on local K8S clusters')
-        if self.cluster.params.get('cluster_backend') == 'k8s-gke':
+    def _k8s_fake_enospc_error(self, node):
+        """Fakes ENOSPC error for scylla container (/var/lib/scylla dir) using chaos-mesh without filling up disk."""
+        if not self._is_chaos_mesh_initialized():
             raise UnsupportedNemesis(
-                'disrupt_nodetool_enospc is skipped due to this: '
-                'https://github.com/scylladb/scylla-cluster-tests/issues/5671')
+                "Chaos Mesh is not installed. Set 'k8s_use_chaos_mesh' config option to 'true'")
+        experiment = IOFaultChaosExperiment(node, duration="120s", error=DiskError.NO_SPACE_LEFT_ON_DEVICE, percent=100,
```

Comment:
What percent mean here ?

Reply:
Probability of failure per operation, in %

Reply:
I copied from chaos-mesh - your'e right - I will rename it to 'error_probability`